fast64_internal/sm64/sm64_objects.py

```diff
@@ -1845,6 +1850,59 @@ def upgrade_changed_props():
             obj.fast64.sm64.version = SM64_ObjectProperties.cur_version
 
 
+class SM64_SegmentProperties(bpy.types.PropertyGroup):
+    group5_seg_str: bpy.props.StringProperty(name="Segment 5 Seg")
+    group5_str: bpy.props.StringProperty(name="Segment 5 Group")
+    group6_seg_str: bpy.props.StringProperty(name="Segment 6 Seg")
+    group6_str: bpy.props.StringProperty(name="Segment 6 Group")
+    group5_enum: bpy.props.EnumProperty(name="Segment 5 Group", default="Do Not Write", items=groupsSeg5)
+    group6_enum: bpy.props.EnumProperty(name="Segment 6 Group", default="Do Not Write", items=groupsSeg6)
```

Comment:
"group5_" sounds like "Group 5" but the name is more like "Group of Segment 5", which is it?

Reply:
It is the group for segment 5. I can rename the things with "group" in them as "seg"

Reply:
`seg5_group`
`seg5_custom_seg`
`seg5_custom_group`
may be good?

Reply:
I chose seg5_load_custom, seg5_group_custom, and seg5_enum